### PR TITLE
es.po + lt.po + fr.po

### PR DESCRIPTION
**Pull request by pr2git**

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -3955,10 +3953,10 @@ msgstr ""
 
 #, python-format
 msgid "FBC socket %s"
-msgstr ""
+msgstr "Conector FBC% s"
```
Comment (nautilus7) [inline po/es.po:3956]: Variable %s is malformed.

File: po/es.po [hunk 2]
```diff
@@ -10988,11 +10980,11 @@ msgstr "Usuario - bouquets"
 
 #, python-format
 msgid "User Band %d"
-msgstr ""
+msgstr "Banda de usuario% d"
 
 #, python-format
 msgid "User Band %d (%s)"
-msgstr ""
+msgstr "Banda de usuario% d (% s)"
```
Comment (nautilus7) [inline po/es.po:10987]: Same here... Variables are malformed.

File: po/es.po [hunk 3]
```diff
@@ -11845,7 +11837,7 @@ msgstr "No ha seleccionado un canal desde el cual grabar."
 
 #, python-format
 msgid "You have to wait %(min)d minutes, %(sec)d seconds!"
-msgstr ""
+msgstr "¡Tienes que esperar %(min)d minutos,% (seg) d segundos!"
```
Comment (nautilus7) [inline po/es.po:11840]: Here as well... It should be %(sec)d like in original string.

Comment (nautilus7) [inline po/es.po:11840]: Here as well,,, It should be %(sec)d like in original string.

File: po/fr.po [hunk 4]
```diff
@@ -7331,7 +7329,7 @@ msgid "Select channel to record from"
 msgstr "Sélectionner la chaîne à enregistrer"
 
 msgid "Select char for replacing newlines in epg."
-msgstr ""
+msgstr "Sélectionnez le caractère pour remplacer les retour à  la  ligne dans epg."
```
Comment (nautilus7) [inline po/fr.po:7332]: Are the double spaces intended here?

File: po/fr.po [hunk 5]
```diff
@@ -10627,7 +10625,7 @@ msgstr "Vous n'avez pas sélectionné une chaîne à enregistrer."
 
 #, python-format
 msgid "You have to wait %(min)d minutes, %(sec)d seconds!"
-msgstr ""
+msgstr "Vous devez attendre %(min)d minutes, %(sec)d secondes !"
```
Comment (nautilus7) [inline po/fr.po:10628]: Is the space before the "!" intended?

**Review by nautilus7 — CHANGES_REQUESTED** [review]

File: po/es.po [hunk 6]
```diff
@@ -10988,11 +10980,11 @@ msgstr "Usuario - bouquets"
 
 #, python-format
 msgid "User Band %d"
-msgstr ""
+msgstr "Banda de usuario% d"
```
Comment (nautilus7) [inline po/es.po:10983]: Same here, malformed variable.

Comment (nautilus7) [inline po/es.po:10983]: Same here, malformed variable.

**Review by nautilus7 — CHANGES_REQUESTED** [review]

**Comment by pr2git** [comment]
The es.po is not mine it is one from angelito thanks for pointing out those errors. Sadly this looks like automatic translation without any check from the creator angelito. I am just creating the PR for him.
I will create another PR just with FR and LT.

**Comment by nautilus7** [comment]
Yes, I know these are not all made by you.

Maybe it's easier to re-open this PR and just correct the errors by adding another commit, or even amend to the initial commit. Isn't it easier to do it this way?

**Comment by nautilus7** [comment]
Or I can fix the es.po and make a PR...

**Pull request closed without merging**